### DISCUSSION TITLE
Latest version of Prettier is not available in Brackets Extension manager

### DISCUSSION THREAD

**Discussion by thresbin** [prettier/prettier#13893] [General]
I'm new to the coding world and found Prettier as I'm starting to explore brackets.  I was having some errors after the install and during my research to fix the errors, I came across this repository and noticed it is much more current than the one found in Brackets.  I would be nice if this version was offered in the extension manager for neophytes like myself.

**Comment by kachkaev** [comment]
👋 @thresbin! [brackets.io](https://brackets.io/) looks somewhat abandoned so it is not very likely that they’ll support newer versions of Prettier. You can try other editors with Prettier integration: https://prettier.io/docs/en/editors.html Good luck!
